jetbrains-core/it/software/aws/toolkits/jetbrains/services/lambda/deploy/SamDeployTest.kt

```diff
@@ -160,7 +163,7 @@ class SamDeployTest {
                                 object : OutputListener() {
                                     override fun onTextAvailable(event: ProcessEvent, outputType: Key<*>) {
                                         super.onTextAvailable(event, outputType)
-                                        println("SAM CLI: ${event.text}")
+                                        LOG.info { "SAM CLI: ${event.text}" }
```

Comment:
This was written this way so the output shows in the gradle logs.

Reply:
how do we fix it so that we get log statements in gradle logs?

Reply:
I THINK we can add a log4j.xml (v1 format) to a test resources and have that log to console.